### ISSUE TITLE
SyntaxError in plugin run-sequence(styles)

### ISSUE THREAD

**Issue by azmeehan** [SnowdogApps/magento2-frontools#346]
We've had frontools up and running with no troubles on a stageing (Mage V 2.2.2 on LAMP) server for about 4-5 months.

Yesterday we tried making it live and we ran composer update which removed our themes.json, after recreating it, now we are running into the following error when running gulp styles.  Wondering if it node/npm/gulp version related, if so I have tried updating to the latest of each but Im still getting this error.  

```
SyntaxError in plugin "run-sequence(styles)"
Message:
    Unexpected end of JSON input
Stack:
SyntaxError: Unexpected end of JSON input
 at JSON.parse (<anonymous>)
    at module.exports (/home/public_html/vendor/snowdog/frontools/helper/config-loader.js:23:19)
    at module.exports (/home/public_html/vendor/snowdog/frontools/helper/scss.js:11:59)
    at themes.forEach.name (/home/public_html/vendor/snowdog/frontools/task/styles.js:17:42)
    at Array.forEach (<anonymous>)
    at Object.module.exports (/home/public_html/vendor/snowdog/frontools/task/styles.js:16:10)
    at module.exports (/home/public_html/vendor/snowdog/frontools/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/public_html/vendor/snowdog/frontools/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/public_html/vendor/snowdog/frontools/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/home/public_html/vendor/snowdog/frontools/node_modules/orchestrator/index.js:134:8)

```

**Comment by Igloczek** [comment]
Error message kinda clearly says that your JSON file is not valid. Could you post your `themes.json` here?
Also please say which version of Frontools you were using before and what is the current version, what config files did you created and where you store them.

Magento, Linux, NPM, Node, Gulp stuff is not important at all, it's just simple JavaScript code.

**Comment by azmeehan** [comment]
OK, so today I ran through the install steps to try and resolve, I notice though when I remove the node modules directory and run npm install I get the following error however I can still run gulp clean (succesfully) and get the original run sequence error when I run gulp styles.  It is possible this happened yesterday could that be causing the original error? If so question is now though what causing this NPM error below?

### themes.json
`{
  "blank": {
    "src": "vendor/realdigi/test-theme",
    "dest": "pub/static/frontend/Realdigi/sass-bootstrap-theme",
    "locale": ["en_AU"]
  }
}`

Using frontools 1.5.16 . possibly 1.5.5 or earlier before.
Node: v8.14.0
Npm: 6.4.1
Gulp: 3.9.1

### Npm Install returns this error
npm install
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://npm.community>

**Comment by Igloczek** [comment]
Fixed in 1.7.0
